frontend/app/components/panel/modal/InviteUser.vue

```diff
@@ -30,9 +30,9 @@ async function onSubmit(event: FormSubmitEvent<Schema>) {
     toast.add({ title: 'Błąd', description: response.message, color: 'error' })
   } else {
     toast.add({ title: 'Sukces', description: 'Pomyślnie zaproszono użytkownika', color: 'success' })
+    open.value = false
+    navigateTo('/panel/team', { external: true })
```

Comment:
Like we are using nuxt for a fucking reason, full page reloads isn't one of the reasons